apps/web/modules/auth/lib/authOptions.test.ts

```
@@ -0,0 +1,285 @@
+  updatedAt: new Date(),
+  twoFactorEnabled: false,
+  identityProvider: "google",
+  objective: "improve_user_retention",
+  notificationSettings: {
+    alert: {},
+    weeklySummary: {},
+    unsubscribedOrganizationIds: [],
+  },
+  role: "other",
+  locale: "en-US",
+};
```

Biome (1.9.4)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)